src/Draggable/Plugins/Announcement/Announcement.js

```diff
@@ -0,0 +1,162 @@
+};
+
+/**
+ * Announcement plugin
+ * @class Announcement
+ * @module Announcement
+ */
+export default class Announcement {
+
+  /**
+   * Announcement constructor.
+   * @constructs Announcement
+   * @param {Draggable} draggable - Draggable instance
+   */
+  constructor(draggable) {
+
+    /**
+     * Draggable instance
+     * @property draggable
+     * @type {Draggable}
+     */
+    this.draggable = draggable;
+
+    /**
+     * Plugin options
+     * @property options
+     * @type {Object}
+     */
+    this.options = {
+      ...defaultOptions,
+      ...this.getOptions(),
+    };
+
+    /**
+     * Original draggable trigger method. Hack until we have onAll or on('all')
+     * @property originalTriggerMethod
+     * @type {Function}
+     */
+    this.originalTriggerMethod = this.draggable.trigger;
+
+    /**
+     * Live region element
+     * @property liveRegion
+     * @type {HTMLElement}
+     */
+    this.liveRegion = createRegion();
+
+    this[onInitialize] = this[onInitialize].bind(this);
+    this[onDestroy] = this[onDestroy].bind(this);
+  }
+
+  /**
+   * Attaches listeners to draggable
+   */
+  attach() {
+    this.draggable
+      .on('draggable:initialize', this[onInitialize]);
+  }
+
+  /**
+   * Detaches listeners from draggable
+   */
+  detach() {
+    this.draggable
+      .off('draggable:destroy', this[onDestroy]);
+  }
+
+  /**
+   * Returns passed in options
+   */
+  getOptions() {
+    return this.draggable.options.announcements || {};
+  }
+
+  /**
+   * Announces event
+   * @private
+   * @param {AbstractEvent} event
+   */
+  [announceEvent](event) {
+    const message = this.options[event.type];
+
+    if (message && (typeof message === 'string')) {
+      this[announceMessage](message);
+    }
+
+    if (message && (typeof message === 'function')) {
+      this[announceMessage](message(event));
+    }
+  }
+
+  /**
+   * Announces message to screen reader
+   * @private
+   * @param {String} message
+   */
+  [announceMessage](message) {
+    const element = document.createElement('div');
+    element.innerHTML = message;
+    this.liveRegion.appendChild(element);
+    return setTimeout(() => {
+      this.liveRegion.removeChild(element);
+    }, this.options.expire);
+  }
+
+  /**
+   * Initialize hander
+   * @private
+   */
+  [onInitialize]() {
+    this.draggable.trigger = (event) => {
+      this[announceEvent](event);
+      this.originalTriggerMethod.call(this.draggable, event);
+    };
+
+    document.body.appendChild(this.liveRegion);
+  }
+
+  /**
+   * Destroy hander
+   * @private
+   */
+  [onDestroy]() {
+    this.draggable.trigger = this.originalTriggerMethod;
+    document.body.removeChild(this.liveRegion);
+  }
+}
+
+/**
+ * Creates region element
+ * @return {HTMLElement}
+ */
+function createRegion() {
+  const element = document.createElement('div');
+
+  element.setAttribute(ARIA_RELEVANT, 'additions');
+  element.setAttribute(ARIA_ATOMIC, 'false');
```

Comment:
I'd suggest changing this to `true` in order to ensure all content is announced. Although, if the content is completely removed from the live region, this shouldn't matter too much.

Reply:
👍  will change this to `true`